chromium_presubmit_config.json5

```diff
@@ -48,7 +48,13 @@
     ],
   },
 
-  // Files to skip in all checks.
+  // Additional files to check (extends InputApi.DEFAULT_FILES_TO_CHECK).
+  "default_files_to_check": [
+    ".+\\.jsx$",
+    ".+\\.tsx$",
+  ],
```

Comment:
And the scripts just know these are frontend files that eslint needs to be run for?

Reply:
> And the scripts just know these are frontend files that eslint needs to be run for?

Not exactly. We run `eslint` with explicit filter that includes these files:
https://github.com/brave/brave-core/blob/149d5ff77e37422781687f7c4de20786ad1fec54/PRESUBMIT.py#L71-L79

This PR extends `InputApi.DEFAULT_FILES_TO_CHECK` which is used by other checks that don't use an explicit filter, such as `CheckFilePermissions`).